server/utils/s3.js

```diff
@@ -0,0 +1,26 @@
+const S3 = require('aws-sdk/clients/s3');
+const fs = require('fs');
+const bucketName = process.env.AWS_BUCKET_NAME
+const region = process.env.AWS_BUCKET_REGION
+const accessKeyId = process.env.AWS_ACCESS_KEY
+const secretAccessKey = process.env.AWS_SECRET_KEY
+
+const s3 = new S3({
+    region,
+    accessKeyId,
+    secretAccessKey
+});
+
+const uploadFile = (file) => {
+    const fileStream = fs.createReadStream(file.path)
```

Comment:
Let's also perform file type validation to only accept image type files

Reply:
Addressed in newest commit at image controller in server